src/classes/CW_AudienceCSVProcessHelperTest.cls

```diff
@@ -0,0 +1,145 @@
+/**
+* File Name   :	CW_AudienceCSVProcessHelperTest.cls
+* Description :	tests CW_AudienceCSVProcessHelper and CSVBatchProess
+*
+* Modification Log
+* ============================================================================
+* Ver Date	   Author		 Modification
+* --- ---------- -------------- --------------------------
+* 0.1 20/05/2014 Alex Panka     Initial version
+
+*
+*/
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.